src/state/gameSlice.ts

```diff
@@ -18,6 +19,11 @@ const gameStateSlice = createSlice({
       // Updating this timestamp tells useCreateGame to perform API call
       state.gameCreationCallTs = Date.now();
     },
+    // dispatch createQuestion when you press start game button
+    createQuestion: (state: GameState, action: Action): void => {
```

Comment:
I recommend renaming this to `fetchQuestion` for clarity

Reply:
done